### DISCUSSION TITLE
Continously Open&Close RTSP connection

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#147] [Q&A]
Hi,
I can see from container logs as attached, bridge keep open RTSP connection at many ports, then gives Error with no one is publishing, then close the connection, and then repeat again withen minute or even seconds, eventhough that specific Camera is streaming fine and intial connecrion was ok (but maybe the wifi signal for that Camera area is not perfect)...any idea why such happen?, and how I can keep the original connection used and no need to open and close many ports...or the stream will change the port and thats why it search all ports?
How it behave?
![Screenshot_20210928-155039_Samsung Internet](https://user-images.githubusercontent.com/89463853/135091547-aad500dd-71dc-4078-b146-b161be6d1eb3.jpg)

**Comment by mrlt8** [comment]
[RTSP] logs come from rtsp-simple-server. I'm guessing you're trying to read a certain path that is not publishing?

**Comment by AdiAbuAli** [comment]
I have one more V3 which is off....let me try to filter and block in environment parameters and check if still trying to read...but I think its related to same camera which has low wifi signal.
Will keep you updated.

Thanks,
________________________________

**Comment by AdiAbuAli** [comment]
Sorry just found the offline V3 is already blacklisted....so most probably from the low wifi signal camera those logs come from

Thanks,
________________________________
